cli/cmd/root.go

```diff
@@ -29,15 +34,30 @@ var rootCmd = &cobra.Command{
 			Label:    "Source Package",
 			Default:  "com.yourcompany",
```

Comment:
Can we get the project name first then set Default for source package to something like "com" + projectNameGot?